src/probnum/diffeq/perturbed/scipy_wrapper/_wrapped_scipy_odesolution.py

```diff
@@ -6,11 +6,11 @@
 from probnum.filtsmooth._timeseriesposterior import DenseOutputValueType
 from probnum.typing import DenseOutputLocationArgType
 
-from .odesolution import ODESolution
+from ..._odesolution import ODESolution
```

Comment:
Correct me if I am wrong, but I thought those relative imports of modules further up in the hierarchy, are discouraged 😬

Reply:
I dont know. I couldnt find anything. Though the google style guide says "use absolute paths all the time". This way we would get syntax like (in `_ivpfiltsmooth.py`)
```python

from probnum import filtsmooth, problems, randprocs, randvars, statespace, utils

from probnum.diffeq import _odesolver
from probnum.diffeq.odefiltsmooth import _kalman_odesolution, initialize
```
If I import ODESolver and KalmanODESolution from probnum.diffeq(.odefiltsmooth), I get circular imports. I am fine with the above though, shall I change it? I think it would clear things up in terms of imports.